python/python/bystro/search/save/handler.py

```diff
@@ -32,7 +32,9 @@
 logger = logging.getLogger(__name__)
 
 MAX_QUERY_SIZE = 10_000
-MAX_SLICES = 1e6
+# TODO 2024-04-26 @akotlar Investigate the impact of large numbers of slices
+# and whether 20_000 slices is a reasonable limit
+MAX_SLICES = 20_000
```

Comment:
20000 slices gives us up to 170 million variant queries before a single slice must scroll. I am not confident there aren't drawbacks to enormous numbers of slices, and we don't actually need 1 slice per 10,000 variants anymore. As the TODO says, we will need to test this.